molecule/_resources/create-docker.yml

```diff
@@ -0,0 +1,84 @@
+---
```

Comment:
Could not the bundled Docker playbook `create.yml` be reused here ?

Reply:
the original playbook puts each host (platform) into a different network, but we need them to communicate. I see no way to pass parameters which would allow that at the moment.

There is also changes for Docker and LXD related to the image name which differs (and is weridly hardcoded for LXD). I'm not sure what proper change I could suggest upstream yet.